src/coreclr/nativeaot/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -832,6 +832,34 @@ REDHAWK_PALEXPORT UInt32_BOOL REDHAWK_PALAPI PalVirtualProtect(_In_ void* pAddre
     return mprotect(pPageStart, memSize, unixProtect) == 0;
 }
 
+REDHAWK_PALEXPORT void PalFlushInstructionCache(_In_ void* pAddress, size_t size)
+{
+#ifndef HOST_ARM
```

Comment:
copy & paste from coreclr pal

Reply:
Is this needed on macOS?

Reply:
HOST_ARM is 32bit ?

Reply:
Looks like HOST_ARM and HOST_ARM64 are mutually exclusive.

https://github.com/dotnet/runtime/blob/76533d4f528a8c51457d9dd0c438f3babbd7733b/src/coreclr/jit/jit.h#L69

Reply:
Yes, host arm is 32-bit.

I have updated the ifdef for clarity.